event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py

```diff
@@ -212,6 +216,9 @@ class BaseProblemCheckTransformer(BaseProblemsTransformer):
     """
     additional_fields = ('result', )
 
+    @openedx_filter(
+        filter_type="event_routing_backends.processors.xapi.problem_interaction_events.base_problem_check.get_object",
```

Comment:
I have some questions related  names of `filter_types`. Other names has the format 
```
...xapi.{particular_word}.{file_name}...
```
Sometimes some have the `particular_word`, other names go directly to `file_name`. Which is the criteria ?
https://github.com/nelc/event-routing-backends/blob/9be505dfdac71bd4ca102cefbbe0afaebb743486/event_routing_backends/processors/xapi/event_transformers/forum_events.py#L18

Reply:
In this case that is wrong that should be `event_routing_backends.processors.xapi.forum_events.base_forum_thread.get_object`